wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WField.java

```diff
@@ -100,10 +100,10 @@ public String getId() {
 	}
 
 	/**
-	 * Finds the first component that can be labelled (if any).
+	 * Finds the first component that can be labeled (if any).
 	 *
 	 * @param component the root of the component hierarchy to check.
-	 * @return the first component that can be labelled, otherwise null.
+	 * @return the first component that can be labeled, otherwise null.

```

Comment:
We probably need a policy on US vs GB spelling...

That said I'll spell "color" with a superfluous "u" when the proverbial freezes over